krabs/krabs/kt.hpp

```diff
@@ -119,11 +119,11 @@ namespace krabs { namespace details {
 
     inline void kt::forward_events(
         const EVENT_RECORD &record,
-        const krabs::trace<krabs::details::kt> &trace)
```

Comment:
We want to keep this `const` everywhere - making the locator const will fix that.